<launch>

    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <rosparam file="$(find qidian_tactics)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find qidian_tactics)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find qidian_tactics)/param/local_costmap_params2.yaml" command="load" />
        <rosparam file="$(find qidian_tactics)/param/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find qidian_tactics)/param/teb_local_planner.yaml" command="load" />
        <rosparam file="$(find qidian_tactics)/param/base_local_planner_params.yaml" command="load" />

        <param name="/move_base/base_local_planner"  value="teb_local_planner/TebLocalPlannerROS" />
    </node>

</launch>
